07 - Arrays/Program.cs

```diff
@@ -5,7 +5,179 @@ namespace _07___Arrays
 {
     class Program
     {
-        
+        static void Main(string[] args)
+        {
+            // Тут многомерный массив с полями
+            string[,] ticTacToe = new string[3, 3]
+            {
+                {"1", "2", "3"},
+                {"4", "5", "6"},
+                {"7", "8", "9"}
+            };
+
+            // это отрисовка его в консоли
+            void ShowDeck()
+            {
+                Console.WriteLine("     |     |     ");
+                Console.WriteLine("  {0}  |  {1}  |  {2}  ", ticTacToe[0, 0], ticTacToe[0, 1], ticTacToe[0, 2]);
+                Console.WriteLine("_____|_____|_____");
+                Console.WriteLine("     |     |    ");
+                Console.WriteLine("  {0}  |  {1}  |  {2}  ", ticTacToe[1, 0], ticTacToe[1, 1], ticTacToe[1, 2]);
+                Console.WriteLine("_____|_____|_____");
+                Console.WriteLine("     |     |     ");
+                Console.WriteLine("  {0}  |  {1}  |  {2}  ", ticTacToe[2, 0], ticTacToe[2, 1], ticTacToe[2, 2]);
+                Console.WriteLine("     |     |     ");
+            }
+
+
+            bool isSquareTaken = false;
+            //тут я храню поле, которое выбрал юзер
+            int chosenSquare;
```

Comment:
why it here, not in ChooseSquare() method?

Reply:
Will it be destroyed as soon as ChooseSquare() is off the stack?